components/esp_lvgl_port/esp_lvgl_port.c

```diff
@@ -23,6 +23,14 @@
 #include "esp_lcd_touch.h"
 #endif
 
+#if __has_include ("usb/hid_host.h")
```

Comment:
Instead of repeating "had_include" a bunch of times, could we define a macro such as "WITH_USB_HID" in CMake if the HID component is present, and then check that simpler macro?

Reply:
I added macros into header file.